Stacks_quickbyte.md

```diff
@@ -0,0 +1,120 @@
+
+Stacks can easily be run on Wheeler with installed modules, and here we outline how with some simple "quality of life" adjustments and tips. We'll be focused on the reference based method, as the non-reference-based is sufficiently run through a driver script provided by the developers of Stacks (denovo_map.pl). We'll quickly mention it at the end. This can often be run on a single node on Wheeler, as the only intense step tends to be alignment, which is quick due to the small size of RADseq data. For example, a dataset of ~90 bird individuals with an average of 1 million reads/sample took four hours on one node. Organisms with larger genomes will take more time and memory.
+
+## Preliminaries ##
+
+### Sample list and Population maps ###
+
+Before you get started, you'll need a list of sample names to run bwa conveniently. This is a file we'll call "sample_list", and it will just look like the following:
+
+	<SAMPLE NAME>
+	M_americana_Florida_MSBBIRD49539
+	M_americana_NM_MSBBIRD39487
+	......
+
+Then, key to many pieces of population genetics software, Stacks needs a population map (popmap) for calculating metrics like F<sub>st</sub> and genetic dviersity. It will also allow the creation of certain imput files. It is simply a tab delimited file with one line per individual, with the first column representing the sample name and the second its population. Note that reduced versions of the popmaps can be made for running _gstacks_ and _populations_ for only a subset of your dataset.
+
+	<SAMPLE NAME>\t<SAMPLE POPULATION>
+	M_americana_Florida_MSBBIRD49539	EastBlackScoter
+	M_americana_NewMexico_MSBBIRD39487	WestBlackScoter
+	..................................	.....
+
+### Demultiplexing with process_radtags ###
+
+Demultiplexing with Stacks is comparatively easy. You just need a file of barcode information (we'll call it BARCODES.file) and your raw, multiplexed reads. The barcode information file can be paired or unapired, and should look like:
+
+	<BARCODE1>\t<BARCODE2>\t<SAMPLE NAME>
+	ATGCAT	GTACGT	M_americana_Florida_MSBBIRD49539
+	ACGTAT	CTAGAT	M_americana_NewMexico_MSBBIRD39487
+	......	......	......
+
+Here is how to run it, assuming you are dealing with paired end reads, gzipped fastq files, and used a single enzyme (ndeI) for your restriction digest. Note the we assume you have a "raw_reads" directory:
+
+	process_radtags -p $src/MULTIPLEXED_READS/ -b $src/BARCODES.file -o $src/raw_reads/ \
+		-i gzfastq -e ndeI -c -q -r -E phred33 
+
+The command is different for single end reads. You must specify each fastq input indiviually with the -f flag, as shown below. The rest is the same:
+
+	process_radtags -f $src/MULTIPLEXED_READS/RAW_READS_01.fastq.gz -b $src/BARCODES.file -o $src/raw_reads/ \
+		-i gzfastq -e ndeI -c -q -r -E phred33
+	process_radtags -f $src/MULTIPLEXED_READS/RAW_READS_02.fastq.gz -b $src/BARCODES.file -o $src/raw_reads/ \
+		-i gzfastq -e ndeI -c -q -r -E phred33
```

Comment:
Here you're referencing some folders and variables you don't make until the next section

Reply:
I decided to refer to them generally (i.e. "/path/to/directory") in a new version I wrote up, and clarified that we'll assume demultiplexing was done before we make the directories/variables.